eng/pipelines/templates/Functional_Tests_On_Windows.yml

```diff
@@ -28,7 +28,7 @@ steps:
   inputs:
     solution: "build\\build.proj"
     configuration: "$(BuildConfiguration)"
-    msbuildArguments: "/MaxCPUCount /ConsoleLoggerParameters:Verbosity=Minimal;Summary;ForceNoAlign /t:RestoreVS /p:BuildNumber=$(BuildNumber) /p:BuildRTM=false /bl:$(Build.StagingDirectory)\\binlog\\01.Restore.binlog"
+    msbuildArguments: "/restore:false /target:RestoreVS /property:BuildNumber=$(BuildNumber) /property:BuildRTM=false /binarylogger:$(Build.StagingDirectory)\\binlog\\01.Restore.binlog"
```

Comment:
In my opinion, the shorter versions of command-line arguments are for humans and any script or YAML should have them spelled out so that people who see them know that they do.  For example, `/bl` is relatively new so not everyone knows what it means.  `/clp` to me isn't obvious that it means "console logger parameters".  Of course you can type whatever you want, I just think that scripts should have the full spelling for novices...  Thoughts?

Reply:
I've never used the full names, even as beginner. Most of the docs use the short versions too.
I get your point about CLP/BL though, t,p are overkills imo, but I don't too care enough to ask you to change it.